canonical-web-and-design/ubuntu.com
# 7339 iot device management

## Done

Added IoT device management takeover and engage page

 [brief](https://docs.google.com/spreadsheets/d/1M5HeoIdOFM8cAJncZF213Ggn8PJ-0Sop-XioQsX1C00/edit#gid=0) and [design](https://github.com/canonical-web-and-design/web-squad/issues/2669)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Refresh until you get the takeover "Secure IoT device management" and follow the link
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)

## Issue / Card

Fixes #7339
